contrib/timelapse.lua

```diff
@@ -0,0 +1,463 @@
+--[[Timelapse video plugin for darktable 2.4.X and 2.6.X
+
+  copyright (c) 2018, 2019  Dominik Markiewicz
+  
+  darktable is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+  
+  darktable is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  
+  You should have received a copy of the GNU General Public License
+  along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[About this plugin
+This plugin will add the new export module "timelapse".
+   
+----REQUIRED SOFTWARE----
+ffmpeg
+
+----USAGE----
+1. Go to Lighttable 
+2. Select images you want to use as a timelapse frames
+3. In image export module select 'timelapse'
+4. Configure you video settings
+5. Export
+
+----WARNING----
+This script has been tested under Linux only
+]]
+
+local dt = require "darktable"
+local df = require "lib/dtutils.file"
+local dsys = require "lib/dtutils.system"
+local gettext = dt.gettext
+
+dt.configuration.check_version(...,{5,0,0})
+
+local MODULE_NAME = 'timelapse'
+gettext.bindtextdomain(MODULE_NAME, dt.configuration.config_dir..'/lua/')
```

Comment:
I would move the OS_PATH_SEPARATOR statement above this, and maybe use PS for the name with a comment specifying what it's for, then code the directory as

dt.configuration.config_dir..PS..'lua'..PS..'locale'..PS

Reply:
done